server/lib/utils.js

```diff
@@ -268,7 +268,7 @@ const unifyItem = (item, type) => {
 
       _.forEach(item.scripts, (script) => { customScripts[script.name] = script.scriptFile; });
 
-      if (item.scripts && item.scripts.length) {
+      if (item.scripts && item.scripts.length && options.enabledDatabaseCustomization !== false) {
```

Comment:
Is there any chance that `options.enabledDatabaseCustomization` would come thru as a string?

Unrelated- using `_.isEmpty(item.scripts)` might be easier than `item.scripts && item.scripts.length`.

Reply:
The extension is getting `options.enabledDatabaseCustomization` from `settings.json`. There can be anything, but it should be `boolean`. Otherwise it won't pass `source-control-tools` validation.